MediaBrowser.Api/Playback/MediaInfoService.cs

```diff
@@ -102,12 +102,6 @@ public class MediaInfoService : BaseApiService
         public object Get(GetBitrateTestBytes request)
         {
             var bytes = new byte[request.Size];
```

Comment:
Can we stop allocating stuff? Or at least hard-cap it to something like 10 megs?

Reply:
I can do both